src/agent/rustjail/src/mount.rs

```diff
@@ -102,6 +102,31 @@ lazy_static! {
     };
 }
 
+#[inline(always)]
+fn mount<P1: ?Sized + NixPath, P2: ?Sized + NixPath, P3: ?Sized + NixPath, P4: ?Sized + NixPath>(
```

Comment:
Dude - you are now a rust guru! :smile:

Reply:
copy and paste guru :sweat_smile: 

https://docs.rs/nix/0.9.0/nix/mount/fn.mount.html